cmd/bridge/main.go

```diff
@@ -229,6 +230,11 @@ func main() {
 		}
 	}
 
+	i18nNamespaces := strings.Split(*fI18NamespacesFlags, ",")
+	if len(i18nNamespaces) == 1 && i18nNamespaces[0] == "" {
```

Comment:
It's an edge case, but does this check fail if the string is nothing but one or more commas? Or if there is only whitespace?